lib/types/application/ApplicationFormatter.js

```diff
@@ -5,6 +5,13 @@ const {promisify} = require("util");
 const readFile = promisify(fs.readFile);
 const AbstractFormatter = require("../AbstractFormatter");
 
+const xml2js = require("xml2js");
```

Comment:
Please move the require and parser initialization to the readPOM function. There is no need to always execute this code.

Reply:
Always running the re-initialization of the parser is a bit more expensive - but I don't care! In general, I agree with the require to be done lazy...

Reply:
Typically the ApplicationFormatter is executed only once anyways.